src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -219,136 +208,13 @@ class ContentSidebar extends PureComponent<Props, State> {
      * @param {boolean} hasFileIdChanged true if the file id has changed
      */
     fetchData(props: Props) {
-        // If nothing to show then just return
-        if (!SidebarUtils.canHaveSidebar(props)) {
-            return;
-        }
-
-        const { fileId, detailsSidebarProps }: Props = props;
-        const { hasAccessStats = false } = detailsSidebarProps;
-        if (!fileId) {
-            return;
-        }
-
-        // Fetch the new file
-        this.fetchFile(fileId);
-
-        if (hasAccessStats) {
-            this.fetchFileAccessStats(fileId);
```

Comment:
Will we need to handle a changed file ID in the details sidebar component now? We only fetch access stats on mount.

Reply:
That should auto happen due to key="fileid" in Sidebar.js
Eventually though, that will have to be replaced by file version id